### PR TITLE
Kartikver15gr8 patch 1

### PR DESCRIPTION
**Pull request by kartikver15gr8**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 :video_camera: Video Resource (Youtube, video course, etc...)
 
 :bulb: Other
-
+
```
Comment (shahednasser) [inline README.md:32]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -529,6 +529,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :bulb: [Using multiple Git accounts with SSH keys](https://gist.github.com/oanhnn/80a89405ab9023894df7)
 - :green_book: [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)
 - :video_camera: [Kunal Kushwaha's Complete Git and GitHub tutorial](https://www.youtube.com/watch?v=apGV9Kg7ics)
+- :video_camera: [Git & GitHub Tutorial For Beginners In Hindi - हिंदी में](https://www.youtube.com/watch?v=gwWKnnCMQ5c&t=2829s)
```
Comment (shahednasser) [inline README.md:532]:
```suggestion
- :video_camera: [Git & GitHub Tutorial For Beginners In Hindi - हिंदी में](https://www.youtube.com/watch?v=gwWKnnCMQ5c)
```

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -29,7 +29,6 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 :video_camera: Video Resource (Youtube, video course, etc...)
 
 :bulb: Other
-
```
Comment (shahednasser) [inline README.md:32]: Keep this empty line but without any spaces

File: README.md [hunk 4]
```diff
@@ -31,7 +31,6 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 :bulb: Other
 
 ## Content
-
```
Comment (shahednasser) [inline README.md:34]: Keep this line as well

**Comment by kartikver15gr8** [comment]
Is it good now

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks! @kartikver15gr8

**Pull request merged**